### PR TITLE
FOR REVIEW: Peg Stability Mechanisms

### PR DESCRIPTION
**Pull request by astronautthis**
Staging links: 
https://makerdao-1.gitbook.io/endgame-docs-staging/concepts/peg-stability-mechanisms
https://makerdao-1.gitbook.io/endgame-docs-staging/endgame/resources
https://makerdao-1.gitbook.io/endgame-docs-staging/endgame/glossary
https://makerdao-1.gitbook.io/endgame-docs-staging/maker-core/list-of-scopes

### CODE REVIEW

File: concepts/peg-stability-mechanisms.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
-# TBC
+# Peg Stability Mechanisms
 
-{% hint style="info" %}
-Sorry! We haven't been able to publish documentation on this topic yet. Please check back soon.
-{% endhint %}
+## Introduction
+
+The Endgame Plan allows the Maker Protocol to dynamically impact the supply and demand of DAI. These mechanisms are called Peg Stability Mechanisms. Note that should not be confused with the similarly-named [Peg Stability Modules (PSMs)](https://manual.makerdao.com/module-index/module-psm)
```
Comment (LongForWisdom) [inline concepts/peg-stability-mechanisms.md:5]: Should change this imo. It implies that these are due to the endgame plan, which is not really true.

Comment (LongForWisdom) [reply concepts/peg-stability-mechanisms.md:5]: Run grammarly on something, second sentence parses strangely.

File: concepts/peg-stability-mechanisms.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+
+{% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
+
+## DAI Savings Rate (DSR)
+
+The [DAI Savings Rate](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) already exists in Maker and remains unchanged in Endgame. It allows DAI holders to deposit DAI into the DSR contract and earn interest. The interest is paid from the protocol's profits.
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply concepts/peg-stability-mechanisms.md:11]: Maybe avoid 'interest' as well. Not sure what the best way to refer to it is, but I suspect it's not this.

File: concepts/peg-stability-mechanisms.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+
+The [DAI Savings Rate](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) already exists in Maker and remains unchanged in Endgame. It allows DAI holders to deposit DAI into the DSR contract and earn interest. The interest is paid from the protocol's profits. 
+
+By changing the interest rate, Maker Governance can increase or decrease the circulating supply of DAI, which in turn influences the price of DAI on the open market.
+
+## Stability Fee Base Rate (SBFR)
+
+The Stability Fee Base Rate increases the stability fee of some or all vaults, increasing income to help fund the DSR. A related and similar parameter that already exists in Maker is the [Global Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee#considerations) parameter.
+
+By changing the stability fee base rate, Maker Governance can make it more or less profitable to mint DAI using vaults. This increases or decreases the circulating supply of DAI, which in turn influences the price of DAI on the open market.
```
Comment (LongForWisdom) [inline concepts/peg-stability-mechanisms.md:19]: Capitalize again here?

Comment (LongForWisdom) [reply concepts/peg-stability-mechanisms.md:19]: More or less expensive rather than profitable, imo.

File: concepts/peg-stability-mechanisms.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+
+## Target Rate (TR)
```
Comment (LongForWisdom) [inline concepts/peg-stability-mechanisms.md:21]: Maybe needs more detail in this section, as it's one of the more unfamiliar concepts to the audience.

Comment (LongForWisdom) [reply concepts/peg-stability-mechanisms.md:21]: Also should maybe highlight the long-term impact of using this lever.

File: concepts/peg-stability-mechanisms.md [hunk 5]
```diff
@@ -1,5 +1,35 @@
...
+
+Certain [Stances](stances.md) in the Endgame Plan allow DAI to depeg from the US Dollar. The Target Rate determines the rate at which the Target Price of DAI changes over time. 
+
+A positive TR increases demand for DAI and reduces the supply of DAI while a negative TR has the opposite effect. 
+
+## Protocol-Owned Vault
+
+The Protocol-Owned Vault is a special Maker Vault only usable by Maker Governance. It holds [EtherDAI](../tokenomics/etherdai.md) and potentially other decentralized assets as collateral. DAI can be minted by this vault. Note that this vault cannot be liquidated and has no Stability Fee.
```
Comment (LongForWisdom) [inline concepts/peg-stability-mechanisms.md:29]: Maybe 'governance can mint Dai against the collateral held in this vault.' Also, I think it might be able to be liquidated, it's just not liquidated automatically.

File: concepts/peg-stability-mechanisms.md [hunk 6]
```diff
@@ -1,5 +1,35 @@
...
+
+Maker Governance may then purchase more EtherDAI with DAI minted from the Protocol-Owned Vault. This puts the protocol in a leveraged long position on EtherDAI and increases the supply of DAI. This action also increases the share of DAI backed by decentralized collateral and takes advantage of a potential negative TR.
```
Comment (LongForWisdom) [inline concepts/peg-stability-mechanisms.md:31]: EtherDAI should definitely be EtherDai, imo. It's not a token symbol. The symbol is ETHD.

Comment (LongForWisdom) [reply concepts/peg-stability-mechanisms.md:31]: 'slightly increases' the share of Dai backed by decentralized collateral.

Comment (LongForWisdom) [reply concepts/peg-stability-mechanisms.md:31]: In practice the numbers are gonna be really small on this.

Comment (LongForWisdom) [reply concepts/peg-stability-mechanisms.md:31]: Maybe we explain the negative TR synergy more here?

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
See comments

File: concepts/peg-stability-mechanisms.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (astronautthis) [reply concepts/peg-stability-mechanisms.md:11]: I've used the term yield and said that it is paid from the system surplus (which can also be negative)

File: concepts/peg-stability-mechanisms.md (hunk 4 above)
Comment (astronautthis) [reply concepts/peg-stability-mechanisms.md:21]: I've addressed this now but is worth a second look/chat before we sign off on it

File: concepts/peg-stability-mechanisms.md [hunk 7]
```diff
@@ -1,5 +1,45 @@
...
+
+Certain [Stances](stances.md) in the Endgame Plan allow Dai to depeg from the US Dollar. The Target Rate determines the rate at which the Target Price of Dai changes over time. 
+
+A positive TR increases demand for Dai and reduces the supply of Dai while a negative TR has the opposite effect.
+
+For example, a TR of -5% implies that the protocol aims to decrease the Target Price of Dai at the rate of 5% per year. 
+
+The TR is an extremely powerful tool as it accomplishes several goals, especially when it is negative.
+ - It enables Maker to withstand a regulatory crackdown that seizes RWA collateral. The target rate may become more negative to reflect a higher risk of RWA collateral being seized.
+ - It encourages Dai generation by ordinary users from decentralized collateral such as EtherDai. This is because the decrease in Dai price offsets the stability fee they must pay.
```
Comment (LongForWisdom) [inline concepts/peg-stability-mechanisms.md:31]: Should be more specific here. It's the expectation that the Dai price will be lower when they want to repay their loan, than when they opened it which incentivizes generation.

File: concepts/peg-stability-mechanisms.md [hunk 8]
```diff
@@ -1,5 +1,45 @@
...
+
+A positive TR increases demand for Dai and reduces the supply of Dai while a negative TR has the opposite effect.
+
+For example, a TR of -5% implies that the protocol aims to decrease the Target Price of Dai at the rate of 5% per year. 
+
+The TR is an extremely powerful tool as it accomplishes several goals, especially when it is negative.
+ - It enables Maker to withstand a regulatory crackdown that seizes RWA collateral. The target rate may become more negative to reflect a higher risk of RWA collateral being seized.
+ - It encourages Dai generation by ordinary users from decentralized collateral such as EtherDai. This is because the decrease in Dai price offsets the stability fee they must pay. 
+
+Despite these benefits, the cost of breaking from the USD peg is also very high. Once the peg has been broken, the selling point that 1 Dai is always worth 1 USD can never be regained.
```
Comment (LongForWisdom) [inline concepts/peg-stability-mechanisms.md:33]: Probably wouldn't use the term selling point here. Maybe confidence / expectation?

File: concepts/peg-stability-mechanisms.md [hunk 9]
```diff
@@ -1,5 +1,45 @@
...
+
+## Protocol-Owned Vault
+
+The Protocol-Owned Vault is a special Maker Vault only usable by Maker Governance. It holds [EtherDai](../tokenomics/etherdai.md) and potentially other decentralized assets as collateral. Governance can mint Dai against the collateral held in this vault. Note that this vault cannot be automatically liquidated and has no Stability Fee.
+
+Maker Governance may then purchase more EtherDai with Dai minted from the Protocol-Owned Vault. This puts the protocol in a leveraged long position on EtherDai and slightly increases the supply of Dai. This action also slightly increases the share of Dai backed by decentralized collateral. 
+
+Finally, one notes that a leveraged long position on EtherDai is more profitable when the value of Dai decreases. Hence, when the Target Rate is negative, the Protocol-Owned Vault can take advantage of this.
```
Comment (LongForWisdom) [inline concepts/peg-stability-mechanisms.md:41]: Maybe reword? "A leveraged long position using the Protocol Owned Vault is synergistic with a negative Target Rate. This is because debt denominated in Dai becomes cheaper to repay as the Target Price of Dai decreases."

File: misc/glossary.md [hunk 10]
```diff
@@ -0,0 +1,156 @@
+# Glossary
+
+
+  
+## Current terms
+
+**Administrative Team:** Team that directly interfaces with SubDAO governance, makes proposals, receives budgets and has governance privileges.
```
Comment (LongForWisdom) [inline misc/glossary.md:7]: I think this is deprecated? Not 100% sure.

Maybe we need sections as: 'Current' 'Probably Deprecated' 'Deprecated' lol.

File: misc/glossary.md [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+
+**Creator DAO:** Type of SubDAO that focuses on decentralized collateral.
+
+**DAI Saving Rate (DSR):** Module that allows users to earn yield from Dai. The term may also refer to the current APY of the DSR module.
+
+**Decentralized Adopted Vault:** Vault with decentralized collateral that only pays a fixed part of their stability fee to Maker Core. A Creator SubDAO is responsible for liquidations and other losses before Maker Core.
+
+**Decentralized Collateral Scope:** Primary Scope that interacts with token asset exchanges. The following tokens are considered decentralized collateral: ETH, MKR, SubDAO, and other decentralized tokens.
+
+**Delegate:** Ecosystem Actor to whom regular MKR holders can safely delegate their MKR voting power without having to hand over the custody of the MKR itself.
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply misc/glossary.md:37]: Should probably add a bunch of links to all of these heh.

File: misc/glossary.md [hunk 12]
```diff
@@ -0,0 +1,156 @@
...
+
+**Elixir:** Balancer pool that consists of ⅓ Dai, ⅓ EtherDai, and ⅓ MKR.
+
+**EtherDai:** Synthetic ETH token backed by staked ETH collateral.
+
+**Facilitator DAO:** Type of SubDAO that focuses on organizing the decentralized workforce of the Maker Core. 
+
+**Finance Scope:** Supporting Scope that interacts with the financial reporting processes and provides updates on the protocol’s financial situation.
+
+**Frontend Service:** External service that aids in the development of publicly accessible aspects of MakeDAO. Examples would be website domain hosting, design, and any other public events.
```
Comment (LongForWisdom) [inline misc/glossary.md:55]: Haven't seen this anywhere. Source?

File: misc/glossary.md [hunk 13]
```diff
@@ -0,0 +1,156 @@
...
+
+**Primary Scopes:** Scope Category that includes Stability and Liquidity, Decentralized Collateral, RWA Collateral, and Ecosystem Operations.
+
+**Protector DAO:** Type of SubDAO responsible for real-world assets.
+
+**Protocol Engineering Scope:** Supporting Scope that is responsible for the functionality and upgrades of the core protocol.
+
+**Protocol-Owned Vault:** Maker vault that can only be utilized by Maker Governance that uses decentralized collateral to generate Dai.
+
+**RWA Adopted Vault:** Vault with RWA collateral that only pays a fixed part of their stability fee to Maker Core. A Protector SubDAO is responsible for liquidations and other losses before Maker Core.
```
Comment (LongForWisdom) [inline misc/glossary.md:97]: ... in exchange for the other part of the fee.

File: misc/glossary.md [hunk 14]
```diff
@@ -0,0 +1,156 @@
...
+
+**RWA Collateral Scope:** Primary Scope that interacts with real world assets that include: cashlike (centralized stable coins & bonds), clean money (renewable energy projects), physical resilient, and miscellaneous (WBTC) assets.
```
Comment (LongForWisdom) [inline misc/glossary.md:99]: physical resilient -> physically resilient

Comment (LongForWisdom) [reply misc/glossary.md:99]: Run through grammarly if you haven't :)

File: misc/glossary.md [hunk 15]
```diff
@@ -0,0 +1,156 @@
...
+
+**RWA Services:** External services that require interaction with the physical world in order to obtain assets.
+
+**Scope:** Broad focus area of work within which SubDAOs, Ecosystem Actors and Maker Governance operate.
+
+**Scope Framework:** Set of rules that SubDAOs within that Scope must adhere to.
```
Comment (LongForWisdom) [inline misc/glossary.md:105]: I think they now cover more than this in practice. Also constraining Maker Core? Maybe more stuff?

File: misc/glossary.md [hunk 16]
```diff
@@ -0,0 +1,156 @@
...
+
+**Supporting Scopes:** Scope Category that includes Protocol Engineering, Growth, Physical Resilence, Infratructure, Interface, and Finance.
+
+**Target Rate (TR):** Goal for the rate at which the target price of Dai changes over time.
+
+**Tribunal:** Entity that is responsible for the creation and adherence to Supporting Scope Frameworks. 
+
+**Unified Coordination Toolkit (UCT):** Open-source software that is utilized as a transparency and governance reporting tool as well as a collaboration tool.
+
+**Unrecognized Delegates:** Delegates that are completely anonymous who do not receive compensation for governance voting.
```
Comment (LongForWisdom) [inline misc/glossary.md:131]: Not sure this is still in use.

File: misc/glossary.md [hunk 17]
```diff
@@ -0,0 +1,156 @@
...
+
+**SubDAO Tokens:** ERC-20 tokens used to participate in SubDAO governance.
+
+**SubElixir:** Liquidity pool consisting of SubDAO tokens and MKR.
+
+**Supporting Scopes:** Scope Category that includes Protocol Engineering, Growth, Physical Resilence, Infratructure, Interface, and Finance.
+
+**Target Rate (TR):** Goal for the rate at which the target price of Dai changes over time.
+
+**Tribunal:** Entity that is responsible for the creation and adherence to Supporting Scope Frameworks.
```
Comment (LongForWisdom) [inline misc/glossary.md:127]: Also not sure this is still in use.

File: misc/glossary.md [hunk 18]
```diff
@@ -0,0 +1,156 @@
...
+
+**Vault Engine:** Regulates the amount of Dai that may be minted by vaults on L2s or vaults operated by SubDAOs.
+
+**Workforce Bonus:** MKR that is utilized to compensate Delegates, Council, and Core Unit members.
+
+**Workforce Bonus Pool:** A pool initially consisting of SubDAO tokens that is distributed based upon SubDAO token governance to incentivize the creation and maintenance of SubDAOs.
+
+
+  
+## Deprecated terms
```
Comment (LongForWisdom) [inline misc/glossary.md:141]: Should include stuff like MetaDAO + MetaElixir

Comment (LongForWisdom) [reply misc/glossary.md:141]: Potentially Elixir as well, but I haven't quite surrendered to that being renamed yet.

**Review by LongForWisdom — CHANGES_REQUESTED** [review]

**Comment by astronautthis** [comment]
Updated based on comments. I didn't start a "likely deprecated" section in the glossary since there weren't enough of them to justify it.

**Review by LongForWisdom — APPROVED** [review]
